scripts/socket_buffer.py

```diff
@@ -18,8 +18,9 @@ def serve():
             try:
                 with lock:
                     for l in  reversed(data):
-                        connection.sendall(l)
-            except:
+                        connection.sendall(l.encode("utf-8"))
```

Comment:
I guess it's ok to disable this pylint error
```suggestion
                        connection.sendall(l.encode("utf-8"))  ## pylint: disable=no-member
```